mrp_workcenter_group/__openerp__.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) All Rights Reserved 2015 Akretion
+#    @author David BEAL <david.beal@akretion.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+
+{
+    'name': 'MRP Workcenter Group',
+    'version': '0.5',
+    'author': 'Akretion,Odoo Community Association (OCA)',
+    'summary': "Organise Workcenters by section",
+    'category': 'Manufacturing',
+    'depends': [
+        'mrp_operations',
+    ],
+    'description': """
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+MRP Workcenter Group
+====================
+
+* Add a new model: Workcenter Groups
+* Add a many2one field 'Group' in workcenter view form based on this model
+* Define a new 'Group by' entry named 'Group' in search view
```

Comment:
Users would rather have this in a human language :) Tell us what it's for, not how it's done.
If we want the technical details we have the code and the ORM's introspection.